tools/run_pr_builder.sh

```diff
@@ -18,7 +18,17 @@ node --version
 # go list is expensive, only do it once.
 GO_LIST="$(go list ./... | grep -v /vendor/ | grep -v /samples/js/)"
 go build ${GO_LIST}
-go test ${GO_LIST}
+
```

Comment:
Unfortunate to have to loop and test but it is required for building the coverage output across multiple golang packages.